sdk/src/main/java/io/horizen/cryptolibprovider/ThresholdSignatureCircuitWithKeyRotation.java

```diff
@@ -30,6 +30,7 @@ Pair<byte[], Long> createProof(List<BackwardTransfer> bt,
                                    Optional<WithdrawalEpochCertificate> previousEpochCertificateOption,
                                    int sidechainCreationVersionNumber,
                                    byte[] genesisKeysRootHash,
+                                   List<byte[]> customFields,
```

Comment:
I've decided to replace the single keyRootHash with a list of custom fields because now we need more than one element in that list, before it was just that one alone with a bunch of zeroes elements.